app-server/app/src/main/resources/logback.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
+    <appender name="file" class="ch.qos.logback.core.FileAppender">
+        <file>logging-app-server.log</file>
```

Comment:
I think this is what you're looking for: http://logback.qos.ch/manual/configuration.html#variableSubstitution